docs/site-replication/README.md

```diff
@@ -56,3 +56,6 @@ mc admin replicate add minio1 minio2 minio3
 ```sh
 mc admin replicate info minio1
 ```
+
+** Note **
+Previously, site replication required root credentials of peer sites be identical. This is no longer required due to STS tokens now being signed with the site replicator service account credentials. However, this means that STS tokens signed previously by root credentials will no longer be valid upon upgrade to the latest version with this change. Also, if site replication is ever removed - the STS tokens previously signed by site replicator service account credentials will no longer be valid.
```

Comment:
```suggestion
Previously, site replication required root credentials of peer sites be identical. This is no longer required due to STS tokens now being signed with the site replicator service account credentials. However, this means that STS tokens signed previously by root credentials will no longer be valid upon upgrade to the latest version with this change. Also, if site replication is ever removed - the STS tokens previously signed by site replicator service account credentials will no longer be valid.
```

We should re-word it to indicate why it is done in this manner, basically to allow the flexibility of having different "root" accounts per site. As well as eventually disabling this root account for all practical purposes.